apps/system/js/app_window_manager.js

```diff
@@ -626,7 +626,12 @@
       this._apps[evt.detail.instanceID] = evt.detail;
     },
 
-    '_handle_homescreen-changed': function() {
+    '_handle_homescreen-changed': function(evt) {
+      if (this.ftuLauncher &&
+          !this.ftuLauncher.respondToHierarchyEvent(evt)) {
```

Comment:
Or looks like we could just do an `if (Service.query('isFtuRunning')) {` instead, probably more readable